src/ui/search-tab.cpp

```diff
@@ -0,0 +1,236 @@
+#include "ui/search-tab.h"
+#include "ui/search-tab-items.h"
+
+#include <QtGlobal>
+
+#if (QT_VERSION >= QT_VERSION_CHECK(5, 0, 0))
+#include <QtWidgets>
+#else
+#include <QtGui>
+#endif
+
+#include "api/requests.h"
+#include "seafile-applet.h"
+#include "account-mgr.h"
+#include "repo-service.h"
+#include "loading-view.h"
+#include "utils/file-utils.h"
+
+namespace {
+const int kMinimumKeywordLength = 2;
```

Comment:
`kMinimumKeywordLength` should be `3` (by default server uses 3 as the minum ngram size)